src/Logging.cpp

```diff
@@ -141,6 +148,11 @@ init(const Config& config)
   }
 #endif
 
+  if (!config.stats_log().empty()) {
+    use_statslog = true;
+    statslog_path = config.stats_log();
```

Comment:
This line won't be reached if `config.log_file() == "syslog"`, so move it before the syslog stuff?